CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ### v4.0.1 Aug 29st, 2024
 
-* Upgrade to Reader SDK 1.7.5 on Android.
+* Upgrade to Reader SDK 1.7.7 on Android.
```

Comment:
I think there should be a new entry to the change log (4.0.2) instead of editing the previous one. And can Aug 29st be fixed also?